src/back/spv/writer.rs

```diff
@@ -585,6 +585,12 @@ impl Writer {
             }
 
             let mut gv = self.global_variables[handle.index()].clone();
+            if let Some(ref mut iface) = interface {
+                // Have to include global variables in the interface
+                if self.physical_layout.version >= 0x10400 {
```

Comment:
Would be nice to do the version conversion later and keep the tuple, but can be another PR.

https://github.com/gfx-rs/naga/blob/c7d02151f08d6285683795289b5725b827d836d1/src/back/spv/writer.rs#L48

Reply:
possible, but not very important. We are already checking for the raw version in another place, so the PR is consistent with the code.